org.lflang/src/org/lflang/generator/ts/TSReactionGenerator.kt

```diff
@@ -333,8 +379,13 @@ class TSReactionGenerator(
                 reactSignatureElement += ": Sched<" + getActionType(effect.variable as Action) + ">"
                 schedActionSet.add(effect.variable as Action)
             } else if (effect.variable is Port){
-                reactSignatureElement += ": ${generateReactionSignatureElementForPortEffect(effect)}"
-                reactEpilogue.add(generateReactionEpilogueForPortEffect(effect))
+                if (reaction.isMutation()) {
```

Comment:
Duplicate code...

Reply:
I deleted the duplicate code. https://github.com/lf-lang/lingua-franca/pull/1318/commits/d58c06655ed025e9537ca8b9dbf5c5af04316984